recipes/forestci/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "forestci" %}
+{% set version = "0.1" %}
+{% set sha256 = "e706ec046c802c2e1036fd8ee306f1d9841b1a9239d0346f8c1849c5966578e0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
It looks like the the `setup.py` requires the import of `numpy` and `scipy`, although it doesn't look they are strictly required to build the package (it looks like they are runtime only reqs).